test/unit/data/prediction_jobs.json

```diff
@@ -35,6 +35,80 @@
             0.7,
             0.9,
             0.95
+        ],
+        "hyperparams" : {
```

Comment:
Same as for the feature comment below. Do we have a clear idea when we use `hyperparams`, hyper_params` or `hyper_parameters` etc.?

Reply:
Futhermore, we might want a more rigid data model perhaps in the future. These hyperparameters seem really specific to xgboost. Just food for thoughts.

Reply:
I suggest we consistently use `hyper_params`.

Reply:
I see wikipedia thinks it should be `hyperparameters`, but I don't mind to use something different. But I think consistency is nice to have. @JanMaartenvanDoorn what is your preference? And should we go for the consistency in this story or someday in the future?
![image](https://user-images.githubusercontent.com/6715707/118001302-e5d5aa80-b346-11eb-9b02-9ac98781bd8d.png)

Reply:
I don't have a strong preference and I am all for consistency. If Hyperparameters is spellen without a '_' I think I would go for 'hyperparameters'.

Reply:
I'm gonna leave this for now, since we already use `hyper_params` in the database and prediction job. Its probably better to go for `hyperparameters` in the future.